README.md

```diff
@@ -162,21 +163,23 @@ A configuration object should conform to the following JSON Schema:
 This sample JSON defines a configuration for the key `express.feature.enabled` with the value _true_ when the _country_code_ is 3, and a default value of _false_.
 
 ```json
-{
-  "alias": "express.feature.enabled",
-  "description": "Feature toggle",
-  "defaultValue": false,
-  "conditions": [
-    {
-      "value": true,
-      "conditionType": {
-        "onValue": "3",
-        "type": "Equals"
-      },
-      "paramName": "country_code"
-    }
-  ]
-}
+[
```

Comment:
I think this listing was always meant as just as an excerpt of a whole file, hence omitting the brackets. I strongly agree with the change though, it has always been confusing :smile: